rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2795,6 +2795,7 @@ def _check_tokens_existence(user_db_cursor: 'DBCursor', expect_removed: bool = F
             receiver=address,
             tx_hash=tx_hash.hex(),  # pylint: disable=no-member
         )
+        write_cursor.execute("INSERT INTO settings(name, value) VALUES ('last_data_upload_ts', 1708294499)")  # setting that will be removed  # noqa: E501
```

Comment:
just edit the DB itself. No need to do it in code everytime the test runs. I have said that multiple times before